eventtracking/django/__init__.py

```diff
@@ -1,179 +0,0 @@
-"""A django specific tracker"""
```

Comment:
Moved all of this code into `django_tracker.py` file to avoid loading at the initializing time and thus the "Models aren't ready yet" error. This tracker will be loaded inside the app's `ready` method.